test/bases.py

```diff
@@ -292,12 +292,12 @@ def test_all_state_can_reach_everything(self):
         """Ensure all state can reach everything and complete the game with the defined options"""
         if not (self.run_default_tests and self.constructed):
             return
-        with self.subTest("Game", game=self.game):
+        with self.subTest("Game", game=self.game, seed=self.multiworld.seed):
             excluded = self.multiworld.worlds[self.player].options.exclude_locations.value
             state = self.multiworld.get_all_state(False)
             for location in self.multiworld.get_locations():
                 if location.name not in excluded:
-                    with self.subTest("Location should be reached", location=location):
+                    with self.subTest("Location should be reached", location=location.name):
```

Comment:
does this keep failing the serialization dumping? i thought we had a proper repr for location, but if we don't i think that's probably the better fix for this

Reply:
This is the cause of the serialization function failing within tests.

Reply:
Looking at it, Location does have a `_repr_` that calls `_str_`. We could use `str(location)` instead, but the end result is the same.